doc/presentations.rst

```diff
@@ -2,6 +2,34 @@
 External Resources, Videos and Talks
 ===========================================
 
+.. _mooc:
+
+The Official scikit-learn MOOC
+==============================
+
+If you are new to scikit-learn, or looking to strengthen your understanding,
+we highly recommend the **scikit-learn MOOC (Massive Open Online Course)**.
+
+The MOOC, created and maintained by some of the scikit-learn core-contributors,
+is **free of charge** and is designed to help learners of all levels master
+machine learning using scikit-learn. It covers topics
+from the fundamental machine learning concepts to more advanced areas like
+predictive modeling pipelines and model evaluation.
+
+The course materials are available at:
+
+`<https://inria.github.io/scikit-learn-mooc/>`_.
+
+This course is also hosted on the `FUN platform
+<https://www.fun-mooc.fr/en/courses/machine-learning-python-scikit-learn/>`_,
+which additionally makes the content interactive without installation, and gives
+access to a discussion forum.
+
+The videos are available as a YouTube playlist on the
+`Inria Learning Lab channel <https://www.youtube.com/@inrialearninglab>`_:
+
+`<https://www.youtube.com/playlist?list=PL2okA_2qDJ-m44KooOI7x8tu85wr4ez4f>`_
```

Comment:
```suggestion
The videos are available on the
`Inria Learning Lab channel <https://www.youtube.com/@inrialearninglab>`_ in
 a `playlist <https://www.youtube.com/playlist?list=PL2okA_2qDJ-m44KooOI7x8tu85wr4ez4f>`_
```

just because the URL is long and not comprehensible